app/controllers/social_networks_controller.rb

```diff
@@ -0,0 +1,32 @@
+class SocialNetworksController < ApplicationController
+  def new
+    @social_network = SocialNetwork.new
+  end
+
+  def create
+    hash = params.require(:social_network).permit(:url, :network)
```

Comment:
Try to call the params in a private method

Reply:
Done